css/style.css

```diff
@@ -0,0 +1,13 @@
+img {
+    max-width: 100%;
+    display: block;
+    height: auto;
+    margin: 0 auto;
+}
+
+/* h1{
+color:aliceblue;
+display: block;
+margin: auto;
+line-height:10px;
+} */
```

Comment:
It is better to not leave unused code as comment 👍. unused code as a comment makes your code not clean ✔.